config.ini

```diff
@@ -65,3 +65,5 @@ far_plane = 300
 keep_perspective = False
 ; show button to visualize related images in a separate window
 show_2d_image = False
+; delete the bounding box after assigning the label to the points
+delete_box_after_assign = True
```

Comment:
Should this be the default?

Reply:
do you mean this should be in the `default_config.ini` or you mean the box should not be deleted by default?